apm-agent-core/src/main/java/co/elastic/apm/agent/util/Version.java

```diff
@@ -72,4 +72,14 @@ public int compareTo(Version another) {
         }
         return 0;
     }
+
+    @Override
+    public String toString() {
+        StringBuilder sb = new StringBuilder();
+        for (int i = 0; i < numbers.length - 1; i++) {
+            sb.append(numbers[i]).append('.');
+        }
+        sb.append(numbers[numbers.length - 1]);
```

Comment:
this breaks if numbers is an empty array

Reply:
Yeah, I didn't care because I only use it for the APM Server, but you're right, if it's here it needs to be safe always